up-subscription/src/subscription_manager.rs

```diff
@@ -509,9 +518,9 @@ async fn remote_unsubscribe(
             make_remote_unsubscribe_uuri(&unsubscribe_request.topic),
             CallOptions::for_rpc_request(
                 UP_REMOTE_TTL,
-                Some(UUID::new()),
+                Some(UUID::build()),
                 None,
-                Some(UPriority::UPRIORITY_CS2),
+                Some(UPriority::UPRIORITY_CS4),
```

Comment:
This will take some time to resolve since the zenoh-transport implementation does not use UNSPECIFIED as the default. When trying to integrate 0.1.1 there are other incompatibilities coming up so it will take some time.

Reply:
In that case, I suggest we stick with explicitly setting CS4 - worst case, we have a bit of extra clarity here, and who knows whether other transports might not also have deviating settings.

Reply:
Ok, I reset it to use CS4 again. Also the InMemoryRpcClient does not handle priority: None messages correctly:

        if let Some(priority) = call_options.priority() {
            builder.with_priority(priority);
        }